src/main/java/lol/client/Client.java

```diff
@@ -37,26 +37,40 @@ public void run() {
     try(Socket s = new Socket(ServerInfo.ip, ServerInfo.port)) {
       socket = s;
       System.out.println("Connection succeeds.");
-      receiveUID();
-      ai.initTeamID(teamID);
-      System.out.println("UID received: " + teamID);
-      Turn turn = ai.championSelect();
-      System.out.println("champion selected");
-      turn.send(socket);
-      System.out.println("sent");
-      allChampionSelection();
-      System.out.println("allCham");
-      allSpawningChampion();
-      System.out.println("Champion selection phase done.");
-      // Now the turn-based game starts until the game is over.
-      arena.startGamePhase();
-      gameLoop();
+      while(1 == 1) {
+        receiveUID();
+        if(teamID >= 2)
+          break;
+        updateEnvironment();
+        ai.initTeamID(teamID);
+        System.out.println("UID received: " + teamID);
+        Turn turn = ai.championSelect();
+        System.out.println("champion selected");
+        turn.send(socket);
+        System.out.println("sent");
+        allChampionSelection();
+        System.out.println("allCham");
```

Comment:
Please, remove the debugging messages (sent, allCham) for me :-) (I forgot to do that).